server/remark-includes.ts

```diff
@@ -425,7 +425,6 @@ export default function remarkIncludes({
       vfile,
       startIndex: lastErrorIndex,
       ruleId: "includes",
-      source: "remark-lint",
```

Comment:
Does naming the source to something beside `remark-lint` also work? I'm curious about which code reads the `source` field.

From a quick browse, it looks like the `remark-message-control` [plugin](https://www.npmjs.com/package/remark-message-control) uses it to allow for disabling/enabling linters via comments (this is used by `remark-lint`). I'm not sure what else uses it.

@iAdramelk, I'm not sure if you're available to answer questions about the `update-vfile-messages.ts` code—if you are, could you provide some context on what this code does? Thanks!

Reply:
> Does naming the source to something beside remark-lint also work? I'm curious about which code reads the source field.

I tried changing it to "something" and that didn't fix it for the include link in the redshift guide.

It does affect the one that 14.x reference that has an ignore comment - if I change `source` to "something" then it's not ignored and I get:
```
content/14.x/docs/pages/contributing/documentation/reference.mdx
    450:1-454:4  error    Wrong import path docs/pages/includes/include.mdx in file content/14.x/docs/pages/contributing/documentation/reference.mdx
```

Otherwise it ignores it correctly.

Having now learned what it's for, I tried this in the redshift serverless file:
```
## Step 3/4. Install and start the Teleport Database Service

{/*lint enable includes*/} 

(!docs/pages/includes/database-access/token.mdx!)
```

But it still doesn't detect the error.

Some more info I learned: in general, if a broken include path is followed by another include path that's valid, and remark-lint is in the source, then it doesn't throw an error, e.g.:
```

(!docs/pages/includes/bad.mdx!)

(!docs/pages/includes/good.mdx!)

```

Inserting text after bad.mdx above will catch the invalid include path.

Reply:
@ptgott AFAIR `source` is only used in reporting. See the right side of this screenshot for example: https://github.com/remarkjs/remark-lint/raw/main/screenshot.png

For my knowledge it should not affect actual linter results in any way.